src/setup/defaultNodes.js

```diff
@@ -48,6 +49,7 @@ const timerEventNodes = [
   [startTimerEvent, 'bpmn:StartEvent', 'bpmn:TimerEventDefinition'],
   [intermediateTimerEvent, 'bpmn:IntermediateCatchEvent', 'bpmn:TimerEventDefinition'],
   [intermediateMessageCatchEvent, 'bpmn:IntermediateCatchEvent', 'bpmn:MessageEventDefinition'],
+  [intermediateMessageThrowEvent, 'bpmn:IntermediateThrowEvent', 'bpmn:MessageEventDefinition'],
```

Comment:
Can you think of a better name for this array? It's currently called `timerEventNodes` but it doesn't only contain timer events...

Reply:
Good point. I put it in with Catch Events, but it was in the timerEventNodes.

Reply:
@JCook21 I used `customEventNodes`. It matches the wording below for `customParserFactory`.